packages/api/src/index.ts

```diff
@@ -99,6 +103,8 @@ passport.use(
           return done(newUserCreationError, null);
         }
       } else {
+        currentUser.githubUsername = profile.username;
+        await authEm?.flush();
```

Comment:
What's the purpose of adding this here?

Reply:
To save the update to the gitHubUsername field above it